### DISCUSSION TITLE
Collections Community Owned Repository

### DISCUSSION THREAD

**Discussion by shmuelie** [dotnet/runtime#42205] [General]
> @Rodrigo-Andrade no, we would take PR's but our time available for it is quite limited. We don't expect to break the API without good reason, if that answers the original question: it should be good for production use. But we may not add more types.
> As for the future -- I would be interested to see a dotnet/collections repo, that is community owned, for any collections that aren't broadly applicable/stable enough to go into the core libraries. That could move more quickly than Microsoft - there's clearly folks passionate about the long tail of collections. Thoughts? Some discussion started here https://github.com/dotnet/corefxlab/pull/2613#issuecomment-665120239

_from: https://github.com/dotnet/corefxlab/issues/2707_

So... how should this work?

**Comment by ericstj** [comment]
Adding some folks to the discussion.  @danmosemsft @eerhardt @richlander @terrajobst

**Reply by danmoseley** [reply]
cc @TylerBrinkley 
area owners @eiriktsarpalis and @layomia
